### PR TITLE
Added system theme as default theme on pageload

### PR DESCRIPTION
**Pull request by Bhavyawahie**
### Closes [https://github.com/AdguardTeam/KnowledgeBase/issues/126](https://github.com/AdguardTeam/KnowledgeBase/issues/126)

> Existing behavior on the [KnowledgeBase page](https://adguard.com/kb/) sets the theme to _Light Theme_ by default and does not adapt to the system-wide theme of the host OS.

### I have made the following tweaks: 

1.  Added a _`colorMode`_ property under _`themeConfig`_ property that has a _`respectPrefersColorScheme`_ which is set to `true` (allowing for system preferred color scheme) in the **`Docusaurus.config.js`**
2. Modified the **`src/css/custom.css`** and added media queries for _`prefers-color-scheme`_ for both light/dark modes.

### CODE REVIEW

**Comment by Bhavyawahie** [comment]
@el-termikael could you please review this PR?

File: package.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
   "private": true,
   "scripts": {
     "docusaurus": "docusaurus",
-    "start": "docusaurus start",
+    "start": "docusaurus start --host 0.0.0.0",
```
Comment (sc0rp10) [inline package.json:7]: I'm not sure about this. Why do we need this change?

File: src/css/custom.css [hunk 2]
```diff
@@ -167,3 +167,11 @@ html[lang='ko'] .navbar__search {
...
   background-image: url("data:image/svg+xml;base64,PHN2ZyBmaWxsPSJub25lIiBoZWlnaHQ9IjE2IiB2aWV3Qm94PSIwIDAgMTcgMTYiIHdpZHRoPSIxNyIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj48ZyBzdHJva2U9IiM2N2IyNzkiIHN0cm9rZS1saW5lY2FwPSJyb3VuZCIgc3Ryb2tlLWxpbmVqb2luPSJyb3VuZCIgc3Ryb2tlLXdpZHRoPSIwLjUiPjxwYXRoIGQ9Im0xMi41IDcuODg4Njd2LTMuODg4ODloLTMuODA5NTIiLz48cGF0aCBkPSJtMTIuNDQ3MyA0LjAyNjE3LTQuNTI2MDUgNC42MjAzMSIvPjxwYXRoIGQ9Im02LjM0NDQ4IDQuNjY2OTljLTEuMDE4NjggMC0xLjg0NDQ4LjgyNTgtMS44NDQ0OCAxLjg0NDQ4djMuNDg4ODNjMCAxLjEwNDYuODk1NDMgMiAyIDJoMy41MzA4Yy45OTU1IDAgMS44MDI1LS44MDcgMS44MDI1LTEuODAyNSIvPjwvZz48L3N2Zz4=");
 }
+
+@media (prefers-color-scheme: light) {
+  /* ... */
+}
+
+@media (prefers-color-scheme: dark) {
+  /* ... */
+}
```
[the remark this answers is not in the document]
Comment (Bhavyawahie) [reply src/css/custom.css:177]: ok, making the required changes

File: package.json (hunk 1 above)
Comment (Bhavyawahie) [reply package.json:7]: Probably not needed, this was done to facilitate me in testing the working of [said change](https://github.com/AdguardTeam/KnowledgeBase/issues/126) on all the devices on my LAN

**Review by sc0rp10 — APPROVED** [review]

**Comment by sc0rp10** [comment]
Thank you @Bhavyawahie for your contribution!
Could you please also apply similar changes in two of our other knowledge bases - [VPN](https://github.com/AdguardTeam/KnowledgeBaseVPN) and [DNS](https://github.com/AdguardTeam/KnowledgeBaseDNS)?
Thanks!

**Comment by Bhavyawahie** [comment]
sure, are there any existing raised issues linked to the same?

**Pull request merged**